### PR TITLE
Added Basic Branding Support

### PR DESCRIPTION
**Pull request by clintonb**
The logo is now easily re-branded. Updated README to include info for additional customization. The Open edX theme is now the default. Below are some examples of customizations for Stanford.

screen shot 2014-11-14 at 2 01 28 pm
screen shot 2014-11-14 at 2 03 46 pm
screen shot 2014-11-14 at 2 09 35 pm
screen shot 2014-11-14 at 2 10 24 pm
screen shot 2014-11-14 at 2 43 01 pm
screen shot 2014-11-14 at 2 49 44 pm

@rocha @dsjen @shnayder @marcotuts @jbau

### CODE REVIEW

**Comment by jbau** [comment]
looks good.  thanks!

File: README.md [hunk 1]
```diff
@@ -118,6 +118,21 @@ Both tools should operate seamlessly in a local development environment. When de
 When creating new pages that utilize RequireJS dependencies, remember to use the `static_rjs` templatetag to load
 the script, and to add a new module to `build.js`.
 
+Theming and Branding
+--------------------
+We presently have support for basic branding of the logo displayed in the header and on error pages. This is facilitated
+by including an additional SASS file specifying the path and dimensions of the logo. The default Open edX theme located
```
Comment (dsjen) [inline README.md]: `SCSS` seems to be used rather than `SASS` in the doc.

Comment (clintonb) [inline README.md]: Updated.

File: analytics_dashboard/static/sass/themes/edx.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+$brand-logo-path: '/static/images/edx-logo-header.png';
+$brand-logo-width: 93px;
+$brand-logo-height: 47px;
```
Comment (dsjen) [inline analytics_dashboard/static/sass/themes/edx.scss:3]: Are these variables used in this file?  Can they be removed?

Comment (clintonb) [inline analytics_dashboard/static/sass/themes/edx.scss:3]: These constants, actually, are specific to the edX logo. They belong here.

**Comment by dsjen** [comment]
Looks good to me!  There's an odd build error it seems.

**Comment by rocha** [comment]
Looks good to me too. :rocket: 

Looks like there is an acceptance test problem.

**Pull request merged**